packages/devtools_app/lib/src/screens/memory/memory_protocol.dart

```diff
@@ -23,51 +23,51 @@ import 'memory_timeline.dart';
 class MemoryTracker {
   MemoryTracker(this.serviceManager, this.memoryController);
 
-  ServiceConnectionManager serviceManager;
+  ServiceConnectionManager? serviceManager;
 
   final MemoryController memoryController;
 
-  VmServiceWrapper get service => serviceManager?.service;
+  VmServiceWrapper? get service => serviceManager?.service;
 
-  Timer _pollingTimer;
+  Timer? _pollingTimer;
 
-  final Map<String, MemoryUsage> isolateHeaps = <String, MemoryUsage>{};
+  final Map<String, MemoryUsage> isolateHeaps = {};
 
   /// Polled VM current RSS.
-  int processRss;
+  int? processRss;
 
   /// Polled adb dumpsys meminfo values.
-  AdbMemoryInfo adbMemoryInfo;
+  AdbMemoryInfo? adbMemoryInfo;
 
   /// Polled engine's RasterCache estimates.
-  RasterCache rasterCache;
+  RasterCache? rasterCache;
 
   bool get hasConnection => service != null;
 
   Stream<void> get onChange => _changeController.stream;
   final _changeController = StreamController<void>.broadcast();
 
-  StreamSubscription<Event> _gcStreamListener;
+  StreamSubscription<Event>? _gcStreamListener;
 
-  Timer _monitorContinues;
+  Timer? _monitorContinues;
 
   void start() {
     _updateLiveDataPolling(memoryController.paused.value);
     memoryController.paused.addListener(_updateLiveDataPolling);
   }
 
-  void _updateLiveDataPolling([bool paused]) {
+  void _updateLiveDataPolling([bool paused = false]) {
```

Comment:
remove paused all together. It is not used in this method

Reply:
done